packages/ws-server/tests/acceptance/estimateGas.spec.ts

```diff
@@ -51,6 +53,26 @@ describe('@release @web-socket eth_estimateGas', async function () {
 
   beforeEach(async () => {
     ethersWsProvider = new ethers.WebSocketProvider(WsTestConstant.WS_RELAY_URL);
+    const wallet = new ethers.Wallet(accounts[0].wallet.privateKey, ethersWsProvider);
```

Comment:
Q: Why do we need to instantiate a new wallet signer while accounts[0].wallet is already a signer?

Reply:
Here we bind the wallet to the `ethersWsProvider`.  This causes the `eth_sendRawTransaction` to run through the `webSocketServer.ts`.  If I optimize this then the deploy calls the `eth_sendRawTransaction` to the RPC server.  This update was to prove that a contract can be deployed via the `webSocketServer.ts`.

Reply:
Ahh yep that makes so much sense! Thanks for clarifying!